internal/commands/sbommonitor/sbommonitor.go

```diff
@@ -26,6 +30,33 @@ func RegisterWorkflows(e workflow.Engine) error {
 	return nil
 }
 
+func loadPolicyFile(policyPath, sbomFilePath string) ([]byte, error) {
+	policy := []byte("\n")
```

Comment:
it's not necessary to have a default of `"\n"`, this can also be empty (i.e. `nil`).